packages/cardpay-reward-programs/streamlit/views/utils.py

```diff
@@ -40,3 +46,23 @@ def download_csv(payment_list):
         file_name="payment_list.csv",
         mime="text/csv",
     )
+
+
+def view_multiple(program):
```

Comment:
These were actually pretty commonly equal across the different programs, and adding more functionality should be broadly applicable.